src/TrackInfo.h

```diff
@@ -33,6 +34,20 @@ namespace TrackInfo
    AUDACITY_DLL_API
    unsigned MinimumTrackHeight();
 
+   //Returns total height of all tracks in the project
+   AUDACITY_DLL_API int MeasureTotalTracksHeight(TrackList& tracks);
+
+   //Returns height of a single channel
```

Comment:
One-line comments for purpose of function should start with
```
//! Brief comment
/*!
Optional longer explanation.
*/
```

So that Doxygen extracts them and formats them nicely.  See doxy.audacityteam.org where this is updated nightly.  See also https://www.doxygen.nl/manual/commands.html.  

An alternative is

```
/*!
@brief Brief comment, maybe multiple lines, then a line break.

Longer explanation.
*/
```